src/main/java/io/mosip/mimoto/controller/IssuersController.java

```diff
@@ -35,10 +36,10 @@ public class IssuersController {
     IssuersService issuersService;
 
     @Autowired
-    private RestApiClient restApiClient;
-
-    private static final String defaultLanguageConstant = "en";
+    IdpService idpService;
```

Comment:
Let's rename `IssuersController` and `IssuersService` to `IssuerController` and `IssuerService`

Reply:
Since the resource is in plural, can we keep the controller and services also in plural